### PR TITLE
BOAC-1330: adds control to change class list page size

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-1330

### CODE REVIEW

File: boac/static/app/course/courseController.js [hunk 1]
```diff
@@ -169,6 +174,17 @@
       });
     };
 
+    $scope.resizePage = function(selectedItemsPerPage) {
+      var currentItemsPerPage = $scope.pagination.itemsPerPage;
+      $scope.pagination.currentPage = Math.round($scope.pagination.currentPage * (currentItemsPerPage / selectedItemsPerPage));
+      $scope.pagination.itemsPerPage = selectedItemsPerPage;
```
Comment (johncrossman) [inline boac/static/app/course/courseController.js]: Nicely done here. The one thing I'd add is:
```
$location.search('page', $scope.pagination.currentPage);
```
That'll keep browser URL current.

File: boac/static/app/course/courseController.js [hunk 2]
```diff
@@ -48,7 +48,12 @@
     $scope.lastActivityDays = utilService.lastActivityDays;
     $scope.pagination = {
       currentPage: 1,
-      itemsPerPage: 50
+      defaultItemsPerPage: 50,
+      itemsPerPage: 50,
+      selected: {
+        itemsPerPage: 50
+      },
+      options: [{label: 'Show 50', value: 50}, {label: 'Show 100', value: 100}]
```
Comment (johncrossman) [inline boac/static/app/course/courseController.js]: @lyttam  Do we expect this value to persist if, for example, I reload the page in my browser?

Comment (johncrossman) [reply boac/static/app/course/courseController.js]: I checked in with the UX dept and the answer is "No", the value does not need to persist. We'll keep the impl simple!

Comment (lyttam) [reply boac/static/app/course/courseController.js]: Sounds good, thanks for checking.

**Pull request merged**